share/completion/fnf

```diff
@@ -0,0 +1,38 @@
+# (C) 2025 unrealapex
+
+# Completion script for fnf
+
+function completion/fnf {
+
+        typeset OPTIONS ARGOPT PREFIX
+        OPTIONS=( #>#
+        "l: --lines:; how many lines to display"
+        "p: --prompt:; input prompt"
+        "s --show-scores; show the scores for each item"
+        "t: --tty:; use tty instead of the default tty device"
+        "q: --query:; use query as the initial search query"
+        "e: --show-matches:; non-interactive mode"
+        "P: --pad:; set left padding of matches"
+        "m: --multi; enable multi-selection"
+        "0: --read-null; read input delimited by ascii null characters"
```

Comment:
The short option names have a `:` flag while the long versions don't. Is this intended?

Reply:
Nope,  thanks for catching that!